tests/sealKey/enc/enc.cpp

```diff
@@ -309,18 +239,18 @@ bool TestOEGetSealKey()
             key_request->cpu_svn,
             0XFF,
             OE_FIELD_SIZE(sgx_key_request_t, cpu_svn));
-        ret = oe_get_seal_key_v1(
-            key_info, key_info_size, third_key_buffer, &third_key_buffer_size);
-        if (ret != OE_INVALID_CPUSVN)
-        {
-            return false;
-        }
         ret = oe_get_seal_key_v2(
-            key_info, key_info_size, &third_key_ptr, &third_key_ptr_size);
+            key_info_ptr,
+            key_info_ptr_size,
+            &third_key_ptr,
+            &third_key_ptr_size);
         if (ret != OE_INVALID_CPUSVN)
         {
             return false;
```

Comment:
if ret is OE_OK then you need to free here too

Reply:
Fixed